src/VuetifyResource.vue

```diff
@@ -237,6 +239,12 @@
           <template slot="no-results">
             {{ lang('no-results') }}
           </template>
+          <template v-for="(_, name) in $slots" v-slot:[name]>
+            <slot :name="name"/>
+          </template>
+          <template v-for="(_, name) in $scopedSlots" :slot="name" slot-scope="slotData">
+            <slot :name="name" v-bind="slotData"/>
+          </template>
```

Comment:
En volgens mij mag alles in $scopedSlots

> All $slots are now also exposed on $scopedSlots as functions. If you work with render functions, it is now recommended to always access slots via $scopedSlots, whether they currently use a scope or not. This will not only make future refactors to add a scope simpler, but also ease your eventual migration to Vue 3, where all slots will be functions.
>> https://vuejs.org/v2/api/#vm-scopedSlots

Dit is de oplossing die @ricardovanlaarhoven en ik laatst hadden. Slots met slotData moet voor slots zonder. Anders gaan (originele scopedSlots niet goed)

```suggestion
          <template v-for="(_, name) in $slots" v-slot:[name]="slotData">
            <slot :name="name" v-bind="slotData"/>
          </template>
          <template v-for="(_, name) in $slots" v-slot:[name]>
            <slot :name="name"/>
          </template>
```

Reply:
Dus de `$scopedSlots` hoeven dan niet worden doorgegeven?

Reply:
Oh foutje in mijn suggestie. Het moet $scopedSlots zijn in plaats van $slots.

Reply:
Als ik de code van jouw suggestie overneem, dan bind die de properties niet meer goed, namelijk bij `<template #expanded-item="{item}">`. Als ik dan `item.message` doe, dan geeft die aan dat het undefined is.